### DISCUSSION TITLE
Got a question about Driwwwle?

### DISCUSSION THREAD

**Discussion by itsnitinr** [itsnitinr/driwwwle#11] [Q&A]
All your queries regarding the codebase and platform itself go here.

**Comment by saroshfarhan** [comment]
Hi @itsnitinr, I am pretty new to Next and React and was having issues while configuring the local setup. While running
`npm run dev` I am getting `MongoParseError: Invalid connection string`. I have added all the files that were there in the README. Do I need to provide any MONGO_URI to the config.env file?
or values to the keys in config.env file?

**Reply by itsnitinr** [reply]
Hi @saroshfarhan 

Yes, you'll need to add the MONGO_URI to config.env

You can either run MongoDB locally on your computer or create a cloud database on MongoDB Atlas to get the connection string. Hope this helps :)

**Reply by saroshfarhan** [reply]
got it, and the same way I need to fill in other keys as well right?

**Reply by itsnitinr** [reply]
Yes!

**Reply by saroshfarhan** [reply]
ok, thank you!!
